### PR TITLE
flakey test: brave.jms.ITJms_2_0_TracingMessageConsumer

### PR DESCRIPTION
**Pull request by reta**
Fixing flakey test: brave.jms.ITJms_2_0_TracingMessageConsumer

### CODE REVIEW

**Comment by reta** [comment]
@codefromthecrypt I was running the test continuously for ~6 hours and it didn't fail for me, wanna give it a try? (and surely, we'll watch for any flakyness).

**Comment by codefromthecrypt** [comment]
@reta so I've never had this fail locally, it has always been on github. Are you saying you have a personal repo running constantly the same test? or on your laptop

File: instrumentation/jms/src/test/java/brave/jms/ArtemisJmsExtension.java [hunk 1]
```diff
@@ -44,6 +44,7 @@ class ArtemisJmsExtension extends JmsExtension {
   ArtemisJmsExtension() {
     factory = new ActiveMQJMSConnectionFactory("vm://0");
     factory.setProducerMaxRate(1); // to allow tests to use production order
+    factory.setReconnectAttempts(3); // to allow tests to reconnect on failure
```
Comment (codefromthecrypt) [inline instrumentation/jms/src/test/java/brave/jms/ArtemisJmsExtension.java:47]: curious what's the default?

**Review by codefromthecrypt — APPROVED** [review]
progress and hopefully works!

**Comment by reta** [comment]
> @reta so I've never had this fail locally, it has always been on github. Are you saying you have a personal repo running constantly the same test? or on your laptop

or on my laptop :)

File: instrumentation/jms/src/test/java/brave/jms/ArtemisJmsExtension.java (hunk 1 above)
Comment (reta) [reply instrumentation/jms/src/test/java/brave/jms/ArtemisJmsExtension.java:47]: Default is `0`

Comment (codefromthecrypt) [reply instrumentation/jms/src/test/java/brave/jms/ArtemisJmsExtension.java:47]: ah good find, likely this will do it! 🤞

**Pull request merged**